Makefile

```diff
@@ -107,7 +112,7 @@ ifndef LLAMA_NO_ACCELERATE
 endif
 ifdef LLAMA_OPENBLAS
 	CFLAGS  += -DGGML_USE_OPENBLAS -I/usr/local/include/openblas
-	LDFLAGS += -lopenblas
+	LDFLAGS += -lopenblas $(LDFLAGS_EXTRA)
```

Comment:
Now thinking about it more. Maybe we should move the whole thing with `ARCH_LINUX := ...` here to make it more local and then use this directly (without the temp variable)?

```
ifdef ARCH_LINUX
LDFLAGS += -lopenblas -lcblas
else
LDFLAGS += -lopenblas
endif
```

Reply:
I've included the check inside of the OPENBLAS def.